Tools/clinic/libclinic/parse_args.py

```diff
@@ -611,40 +661,39 @@ def parse_general(self, clang: CLanguage) -> None:
             use_parser_code = False
             self.fastcall = False
         else:
-            if self.vararg == NO_VARARG:
-                self.codegen.add_include('pycore_modsupport.h',
-                                         '_PyArg_UnpackKeywords()')
-                args_declaration = "_PyArg_UnpackKeywords", "%s, %s, %s" % (
-                    self.min_pos,
-                    self.max_pos,
-                    self.min_kw_only
-                )
+            self.codegen.add_include('pycore_modsupport.h',
+                                     '_PyArg_UnpackKeywords()')
+            if not self.varpos:
                 nargs = "nargs"
             else:
-                self.codegen.add_include('pycore_modsupport.h',
-                                         '_PyArg_UnpackKeywordsWithVararg()')
-                args_declaration = "_PyArg_UnpackKeywordsWithVararg", "%s, %s, %s, %s" % (
-                    self.min_pos,
-                    self.max_pos,
-                    self.min_kw_only,
-                    self.vararg
-                )
                 nargs = f"Py_MIN(nargs, {self.max_pos})" if self.max_pos else "0"
 
             if self.fastcall:
                 self.flags = "METH_FASTCALL|METH_KEYWORDS"
                 self.parser_prototype = PARSER_PROTOTYPE_FASTCALL_KEYWORDS
-                argname_fmt = 'args[%d]'
                 self.declarations = declare_parser(self.func, codegen=self.codegen)
                 self.declarations += "\nPyObject *argsbuf[%s];" % len(self.converters)
+                if self.varpos:
+                    self.declarations += "\nPyObject * const *fastargs;"
```

Comment:
```suggestion
                    self.declarations += "\nPyObject *const *fastargs;"
```

Reply:
`const` without preceding space looks weird. I think the code is clearer this way.

Reply:
Ok, so actually, in the function signature, it appears that `PyObject *const *args` is used *but* in the function *body*, it appears that we only have `PyObject * const *fastargs` (there are no other usage of PyObject * const *... in a function body). Up to you.

Reply:
Please leave it as it is; there is no need to increase the diff for such unneeded stylistic changes.